packages/odyssey-storybook/src/utilities/Utilities.stories.mdx

```diff
@@ -0,0 +1,202 @@
+
+```html
+<Text>
+  <blockquote>
+    <p>
+      The important thing is not to stop questioning. Curiosity has its own
+      reason for existence.
+    </p>
+    <footer>
+      Albert Einstein,
+      <cite>
+        Old Man's Advice to Youth: "Never Lose a Holy Curiosity," LIFE magazine
+        (2 May 1955) statement to William Miller, p. 64.
+      </cite>
+    </footer>
+  </blockquote>
+</Text>
+```
+
+## Current Cases
+
+Currently, we have a few open PRs that stand as practical examples for application of these utilities. Below, I've outlined my current understanding of what each component should look like, given these utilities.
+
+### Field
+
+```html
+// Field
+
+<Box as="{tag}" className="{styles.root}">
+  <Field.Label
+    inputId="{inputId}"
+    required="{required}"
+    optionalLabel="{optionalLabel}"
+    labelHidden="{labelHidden}"
+    as="{TagLabel}"
+  >
+    {label}
+  </Field.Label>
+  {hint && <Field.Hint id="{inputId}">{hint}</Field.Hint>} {children} {error &&
+  <Field.Error id="{inputId}">{error}</Field.Error>}
+</Box>
+
+// Label //// In the case of these label elements, we can treat our Field
+styling as the HTML5 default, removing the need for component-specific styling.
+
+<Text as="{labelTag}" htmlFor="{inputId}">
+  {children} {!required && optionalLabel && (
+  <span className="{styles.optionalLabel}" children="{optionalLabel}" />
+  )}
+</Text>
+
+// Hidden Label
+
+<ScreenReaderText>
+  <Text as="{labelTag}" htmlFor="{inputId}">
+    {children} {!required && optionalLabel && (
+    <span className="{styles.optionalLabel}" children="{optionalLabel}" />
+    )}
+  </Text>
+</ScreenReaderText>
+
+// Hint
+
+<Box
+  as="p"
+  className="{styles.hint}"
+  id="{`${id}-hint`}"
+  children="{children}"
+/>
+
+// Error
+
+<Box as="p" className="{styles.error}" id="{`${id}-error`}">{children} /> </Box>
+```
+
+### Heading
+
+Much like `<Text />` is a convenience wrapper on top of `<Box />`, `<Heading />` is built on `<Text />`.
+
+```html
+// Heading
+
+<Text as="{tag}" className="{componentClass}"> {children} </Text>
+```
+
+My assumption here is that all Headings have parity with our desired HTML5 defaults. The `className` would be used to apply a different level of visual hierarchy to a tag (an `h1` with the styles of an `h3`).
+
+If a component requires heading styling that deviates farther than that, those styles should live with the component until they become a pattern.
+
+### List
+
+```html
+// List
+
+<Box as="{tag}"> { children } </Box>
+
+//// `tag` here would be ul, ol, dl // ListItem
+
+<Text as="{childTag}"> { children } </Text>
+```
+
+If `<List>` were only intended to be used in the fashion of plain HTML5, `<Text>` could probably be used throughout. e.g. `<Text as="ul">`. However, we've seen that `<List>` is likely to have more complicated functionality.
+
+Alternately, using `<Text as="ul">` could provide default styling while `<List>` comes without assumed margins and padding.
```

Comment:
I'm not sure I understand the doc writeup here or the code example. My understanding was the List already has 99% of what it needs, just missing the ability for children passed in to be styled via inheritance which Box will provide if placed at the root of List. That is an implementation detail that doesn't really need to be documented

Reply:
The use cases for lists (generally) speak to different underlying needs:

If I am styling dynamic content or simply want to insert a `ul` into my page, I would expect it to have margin, padding, and little dots.

If I am assembling Action List, which is semantically a `ul` but not visually, I would expect List to be free from the visual standards assumed in the first case.

My suggestion here is that the former case would be better off using `<Text as="ul">` or nesting a `ul` inside of `<Text>`, whereas the latter case is looking for `<List>`.